packages/cypress/package.json

```diff
@@ -23,6 +23,8 @@
       "import": "./dist/support.mjs",
       "require": "./dist/support.js"
     },
+    "./bin/archive-storybook": "./dist/bin/archive-storybook.js",
```

Comment:
@tmeasday do these need the `bin/`, or should these just be like `"./archive-storybook": "./dist/bin/archive-storybook.js"`?

Reply:
Since the CLI references these with the `bin` in the path, we should leave it as is.